lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -888,3 +900,45 @@ class _EditableSizeAndTransform {
   final double height;
   final Float64List transform;
 }
+
+/// Used for detecting the changes in the selection.
+///
+/// Currently only used in Firefox.
+///
+/// In Firefox, when cursor moves, nor selectionChange neither onInput
```

Comment:
Let's be clear that it's about _text_ cursor, not any other kind of cursor.

Reply:
done.